src/components/PopupWithForm.js

```diff
@@ -16,20 +26,40 @@ class PopupWithForm extends Popup {
     return formValues;
   }
 
+  open() {
+    super.open();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.